### ISSUE TITLE
[Markup and meta data] HTML markup doesn't parse correctly. (09.03.1)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#66637]
#### VFS team name
MHV

#### VFS product name
MHV Medical Records

#### VFS feature name
Allergies

#### Point of Contact/Reviewers
Brian DeConinck - @briandeconinck - Accessibility

*For more information on how to interpret this ticket, please refer to the [Anatomy of a Staging Review issue ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Anatomy-of-a-Staging-Review-Issue-ticket.2060320997.html) guidance on Platform Website.

---

### Platform Issue
HTML markup doesn't parse correctly.

### Issue Details
Both /medical-records/allergies and /medical-records/allergies/:allergyId contain HTML elements with duplicate IDs. Looks like this is happening with the hidden print-only content, eg. there's an `#allergy-date` that's visible on the page and another `#allergy-date` that's not visible and is contained in the `.print-only` div. The `.print-only` div is hidden with `display: none` which should hide it from just about all assistive technologies, so the impact is minimal. But there may be some users running *very* old screen readers that get tripped up by having duplicate IDs on the page.

### VA.gov Experience Standard
[Category Number 09, Issue Number 03](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Other References
WCAG SC 4.1.1_A

### Platform Recommendation
This is low priority to resolve, because all of the `print-only` content is hidden from assistive tech and won't impact the vast majority of users. But just for safety, giving the print version elements an ID prefix would be an easy solution, eg. `id="print-allergy-date'`.

---

### VFS Guidance

- Close the ticket when the issue has been resolved or validated by your Product Owner
- If your team has additional questions or needs Platform help validating the issue, please comment on the ticket
- Some feedback provided may be out of scope for your iteration of the product, however, Platform's OCTO leadership has stated that all identified issues need to be documented and it is still your responsibility to resolve the issue.
- If you do not believe that this Staging Review issue ticket is the responsibility of your team, comment below providing an explanation and who you believe is responsible. Please tag the Point of Contact/Reviewers. Governance team will research and will follow up.

**Comment by BobbyBaileyRB** [comment]
Complete
